Sources/MuxUploadSDK/PublicAPI/MuxUpload.swift

```diff
@@ -404,8 +404,7 @@ public final class MuxUpload : Hashable, Equatable {
      the upload has been prepared
      */
     public var videoFile: URL? {
-        #warning("This isn't the actual implementation, should return either the exported asset file URL or the standardized file URL depending on the inspection result")
-        return (input.sourceAsset as? AVURLAsset)?.url
+        return fileWorker?.inputFileURL
```

Comment:
question(blocking): The original warning called out an "or", but that isn't obviously captured in this implementation change, particularly given the naming of `fileWorker.inputFileUrl`. Mind explaining what happened there?

Reply:
Good question, `fileWorker` is initialized with the correct file URL when the upload is done preparing.

Reply:
cool that's what I was hoping/assuming. So `inputFileURL` may be slightly confusing since it may actually be the "standardized file URL". Not requesting a change here, but food for thought.

Reply:
`input` is a reference to the [Mux Video defined term](https://docs.mux.com/api-reference#video/operation/create-direct-upload) for what we actually send to the direct upload. 

I'd prefer to keep internal SDK terminology consistent with Mux Video to the extent possible so it's easy for those new to the codebase to navigate it.

Reply:
Additionally, whether the file is standardized or not is irrelevant to `ChunkFileUploader`.